apache/superset
# feat(helm): support different replicaCount

### SUMMARY
We want to configure a different replicaCount for `supersetNode` and `supersetWorker` deployments, as they do not undergo the same load.

The `replicaCount` should be configurable on a per component level instead of globally for the whole chart.

### TESTING INSTRUCTIONS
Templating can be verified manually with `helm template`
- `helm lint`
- `helm3 template superset-release .`
- Check that the new values are now required
```
❯ helm template superset-release .  --set supersetWorker.replicaCount=null --set supersetNode.replicaCount=null

Error: values don't meet the specifications of the schema(s) in the following chart(s):
superset:
- supersetWorker: replicaCount is required
- supersetNode: replicaCount is required
```

### ADDITIONAL INFORMATION


- [x] Has associated issue: #19250
